index.bs

```diff
@@ -795,6 +795,89 @@ Sequence:
   </pre>
 </div>
 
+
+## Period datatype ## {#period-datatype}
+
+<pre class='railroad'>
+Choice:
+  Sequence:
+    N: start
+    T: /
+    N: end
+  Sequence:
+    N: start
+    T: /
+    N: duration
+  Sequence:
+    N: duration
+    T: /
+    N: end
+  T: duration
+</pre>
+
+Where "start" and "end" are [[#datetime-datatype]] and "duration" is defined
+as:
+
+<pre class='railroad'>
+T: P
+OneOrMore:
+  Stack:
+    Sequence:
+      Sequence:
+        Optional:
+          Sequence:
+            N: 0-9
+            N: 0-9
+            N: 0-9
+            N: 0-9
+            T: Y
+        Optional:
+          Sequence:
+            N: 0-9
```

Comment:
Does this mean that you cannot specify 10 months? I'm wondering why there are not 2 numbers permitted here.

Reply:
Very good point :) I'll review.